ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -1,39 +1,37 @@
 #!/bin/bash
 
 if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
-    source "${SHARED_DIR}/proxy-conf.sh"
+  source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
 pwd && ls -ltr
 cd frontend || exit 0
 pwd && ls -ltr
 
 ## skip all tests when console is not installed
-ret=0
-oc get co console --kubeconfig=${KUBECONFIG} || ret=$?
-if [ $ret -ne 0 ]; then
+if (oc get clusteroperator console --kubeconfig=${KUBECONFIG}) ; then
   echo "console is not installed, skipping all console tests."
   exit 0
 fi
 
-export E2E_RUN_TAGS="${E2E_RUN_TAGS}"
-echo "E2E_RUN_TAGS is '${E2E_RUN_TAGS}'"
-## determine if running against managed service or smoke scenarios
-if [[ ($E2E_RUN_TAGS == *"@osd_ccs"*) || ($E2E_RUN_TAGS == *"@rosa"*) ]]; then
-  echo "Testing against online cluster"
-  ./console-test-osd.sh || exit 0
-elif [[ $E2E_RUN_TAGS == *"@smoke"* ]]; then
-  echo "only run smoke scenarios"
-  ./console-test-frontend.sh --tags @smoke || exit 0
-fi
-
 ## determine if it is hypershift guest cluster or not
-res=0
-oc get node --kubeconfig=${KUBECONFIG} | grep master || res=$?
-if [ $res -eq 0 ]; then
-  echo "Testing on normal cluster"
-  ./console-test-frontend.sh || exit 0
-else
+if ! (oc get node --kubeconfig=${KUBECONFIG} | grep master) ; then
   echo "Testing on hypershift guest cluster"
-  ./console-test-frontend-hypershift.sh || exit 0
+  ./console-test-frontend-hypershift.sh || true
+else
+  export E2E_RUN_TAGS="${E2E_RUN_TAGS}"
+  echo "E2E_RUN_TAGS is: ${E2E_RUN_TAGS}"
+  ## determine if running against managed service or smoke scenarios
+  if [[ $E2E_RUN_TAGS =~ @osd_ccs|@rosa ]] ; then
+    echo "Testing against online cluster"
+    ./console-test-osd.sh || true
+  # if the TAGS contains @console, then it's a job specific for UI, run full tests
+  # or else, we run smoke tests to balance coverage and cost
+  elif [[ $E2E_RUN_TAGS =~ @console ]]; then
```

Comment:
@yanpzhan Could you help to double-confirm this label @console, as we didn't have this label currently in the test case, I have some confuse here
cc: @liangxia

Reply:
The label `@console` comes from UI specific job in openshift/release repo, [example](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml#L111-L122)
For smoke tests in cypress, it is executing `./console-test-frontend.sh --tags @smoke`. The `@smoke` used here comes from UI test cases.